MVP/defaults-build.yaml

```diff
@@ -1,6 +1,23 @@
 - defaults:
     name: defaults-build
     node: 'jslave-platform-rhel7'
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: tag
```

Comment:
Should default values and/or a description be added for each parameter then?  It may help the user understand what is required to be filled out when triggering a manual build.